lib/bolt/outputter/human.rb

```diff
@@ -101,7 +101,7 @@ def print_result(result)
         end
 
         # There is more information to output
-        if result.generic_value
+        unless result.generic_value.nil? || result.generic_value.empty?
```

Comment:
I think we need to always output _something_. So if there's no `_error` or `_output`, we should print the generic value even if it's empty.